xds/src/test/java/io/grpc/xds/XdsSdsClientServerTest.java

```diff
@@ -103,16 +108,17 @@ public void tearDown() {
   }
 
   @Test
-  public void plaintextClientServer() throws IOException, URISyntaxException {
+  public void plaintextClientServer() throws Exception {
     buildServerWithTlsContext(/* downstreamTlsContext= */ null);
 
     SimpleServiceGrpc.SimpleServiceBlockingStub blockingStub =
-        getBlockingStub(/* upstreamTlsContext= */ null, /* overrideAuthority= */ null);
+        getBlockingStub(/* upstreamTlsContext= */ null,
+                /* overrideAuthority= */ OVERRIDE_AUTHORITY);
```

Comment:
for plaintext we don't need `OVERRIDE_AUTHORITY` so why was this changed from `null`?